vectors_deep1b/operations/default.json

```diff
@@ -0,0 +1,19 @@
+{       
+  "name": "index-append",
+  "operation-type": "bulk",
+  "bulk-size": {{bulk_size | default(5000)}},
+  "ingest-percentage": {{ingest_percentage | default(100)}}
+},
+{
+  "name": "knn_search",
```

Comment:
I don't see this operation referenced anywhere in the challenge. Do you intend to add it later?

Reply:
Yes, we want to add this operation as soon as it becomes [available in rally](https://github.com/elastic/rally/issues/1380) .   `_knn_search`  is a new API in elasticsearch, and has not been implemented in rally yet. Speaking of which, would it be possible for the benchmarking team to implement it, or should our search team do it?

Reply:
> Yes, we want to add this operation as soon as it becomes [available in rally](https://github.com/elastic/rally/issues/1380) . `_knn_search` is a new API in elasticsearch, and has not been implemented in rally yet. Speaking of which, would it be possible for the benchmarking team to implement it, or should our search team do it?

@mayya-sharipova Thanks for the link! I see in the docs that this is an experimental API and might change. If that's true, do you think it makes more sense to use [raw-request](https://esrally.readthedocs.io/en/latest/track.html?highlight=raw-request#raw-request) or a [custom runner](https://esrally.readthedocs.io/en/latest/adding_tracks.html?highlight=custom%20runner#custom-runners) to issue this request?

Reply:
Great question, considering that it is an experimental, and in the future we may move it under `_search`, I guess it makes sense to have a raw request for now.